app/Importer/LicenseImporter.php

```diff
@@ -73,12 +73,21 @@ public function createLicenseIfNotExists(array $row)
                 $asset = Asset::where('asset_tag', $asset_tag)->first();
                 $targetLicense = $license->freeSeat();
```

Comment:
This `$targetLicense` really probably ought to be called `$targetSeat` or something like that? Because it's not a license, it's a seat.

Reply:
I rename the variable, you are right I don't named the variable originally but this kind of things need to got my attention.